sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
-
+- Entity Name is handled properly when passed in as part of connection string in `ServiceBusClientBuilder`
```

Comment:
Do we generally use a `Fixed <something>` in bugs fixed section?

Reply:
I am not sure if there are standards around how to word the change entry into changelog, I guess we just go with what best decribes the change :)